deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -840,6 +817,619 @@ spec:
         #    path: /var/cache/Elastic
         #    type: DirectoryOrCreate
 ---
+# Configmap for Daemoset Agents. It includes all the datastreams except Kube-State-metrics
```

Comment:
Why is this added here? Looks weird and kind of irrelevant to the PR?

Reply:
Removed by correcting the makefile and rerun